src/test/java/org/trebol/jpa/services/conversion/SellStatusesConverterJpaServiceImplTest.java

```diff
@@ -0,0 +1,70 @@
+package org.trebol.jpa.services.conversion;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.core.convert.ConversionService;
+import org.trebol.exceptions.BadInputException;
+import org.trebol.jpa.entities.SellStatus;
+import org.trebol.pojo.SellStatusPojo;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+import static org.trebol.constant.TestContants.ANY;
+
+@ExtendWith(MockitoExtension.class)
+class SellStatusesConverterJpaServiceImplTest {
+    @InjectMocks
+    private SellStatusesConverterJpaServiceImpl sut;
+
+    @Mock
+    private ConversionService conversionService;
+
+    private SellStatus sellStatus;
+    private SellStatusPojo sellStatusPojo;
+
+    @BeforeEach
+    void beforeEach() {
+        sellStatus = new SellStatus();
+        sellStatus.setName(ANY);
+        sellStatus.setId(1L);
+        sellStatus.setName(ANY);
+
+        sellStatusPojo = new SellStatusPojo();
+        sellStatusPojo.setName(ANY);
+    }
+
+    @AfterEach
+    void afterEach() {
+        sellStatus = null;
+        sellStatusPojo = null;
+    }
+
+    @Test
+    void testApplyChangesToExistingEntity() throws BadInputException {
+        sellStatusPojo.setName("PIOLO");
+        SellStatus actual = sut.applyChangesToExistingEntity(sellStatusPojo, sellStatus);
+        assertEquals(1L, actual.getId());
```

Comment:
This assertion has nothing to do with what's being done in the method. Perhaps testing the equality of "PIOLO" vs `actual` name?